### PR TITLE
Changing ProductInstance interface when loading and finding

### PR DESCRIPTION
**Pull request by jmmovilla**
*** REVIEWERS ***
@hmunfru @flopezag 

*** DESCRIPTION ***
Changing retunr type in loads and finds methods

### CODE REVIEW

File: client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/BaseInstallableService.java [hunk 1]
```diff
@@ -80,7 +80,7 @@
      * @throws ResourceNotFoundException
      *             if the resource does not found
      */
-    T load(String vdc, String name, String token) throws ResourceNotFoundException;
+    //T load(String vdc, String name, String token) throws ResourceNotFoundException;
```
Comment (hmunfru) [inline client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/BaseInstallableService.java]: why are you removing the interfaces in the base service?

File: client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/ProductInstanceService.java [hunk 2]
```diff
@@ -26,14 +26,14 @@
...
 import com.telefonica.euro_iaas.sdc.model.Artifact;
 import com.telefonica.euro_iaas.sdc.model.InstallableInstance.Status;
-import com.telefonica.euro_iaas.sdc.model.ProductInstance;
+import com.telefonica.euro_iaas.sdc.model.Attribute;
 import com.telefonica.euro_iaas.sdc.model.Task;
 import com.telefonica.euro_iaas.sdc.model.dto.ProductInstanceDto;
 
-public interface ProductInstanceService extends BaseInstallableService<ProductInstance> {
-
+public interface ProductInstanceService {
```
Comment (hmunfru) [inline client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/ProductInstanceService.java]: why are you removing the baseinstallableservice?

**Comment by hmunfru** [comment]
No merge till the PaaS Manager change has been introduced

File: client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/BaseInstallableService.java (hunk 1 above)
Comment (jesuspg) [inline client/src/main/java/com/telefonica/euro_iaas/sdc/client/services/BaseInstallableService.java]: please, no comment the code; remove it

**Pull request merged**